test/test15.c

```diff
@@ -6,33 +6,44 @@
 
 #define Annotate(x) __attribute__((annotate(x)))
 
-Annotate("DoesFree")
+#define MustCall(x) __attribute__((annotate("MustCall: {" #x "}")))
+#define CalledMethods(x) __attribute__((annotate("CalledMethods: {" #x "}")))
+#define MustCallUnkown __attribute__((annotate("MustCallUnknown: {}")))
+
+// unsure how much we'll be needing these 
+#define Owning __attribute__((annotate("Owning")))
+#define MustCallAlias __attribute__((annotate("MustCallAlias"))) // implementation will be tricky  since this needs a pair of annotations, one on the return type and one for a parameter
```

Comment:
These should definitely be in their own header file that a client program (like a test) should include rather than in the tests directly. For now, this is probably fine, but we'll need to do that eventually so if it's easy to do you should do it now.